notebooks/1_LinearMethods.ipynb

```diff
@@ -1306,13 +1306,13 @@
    "source": [
     "data = chemiscope_input(\n",
     "    # Dataset name\n",
-    "    meta=dict(name=\"CSD-1000R\"),\n",
+    "    meta={'name':\"CSD-1000R\"},\n",
```

Comment:
@Luthaf might be interested in:
https://stackoverflow.com/questions/17097985/dict-vs-in-python-which-is-better/17098017

Reply:
I tend to prefer `{}` as well, you can have keys which are not valid identifiers: `{"a long key": 3}` =)